sklearn/ensemble/weight_boosting.py

```diff
@@ -70,16 +75,19 @@ def __init__(self,
         self.learning_rate = learning_rate
         self.random_state = random_state
 
+    def _validate_data(self, X):
+        return check_array(X,
+                           accept_sparse=True,
+                           ensure_2d=False,
+                           allow_nd=True)
+
     def fit(self, X, y, sample_weight=None):
         """Build a boosted classifier/regressor from the training set (X, y).
 
         Parameters
         ----------
         X : {array-like, sparse matrix} of shape = [n_samples, n_features]
-            The training input samples. Sparse matrix can be CSC, CSR, COO,
```

Comment:
Is this no longer true of the default base estimator at least

Reply:
I revert back the docstring